### ISSUE TITLE
Why vct is not selectively disclosable, but vct#integrity is not?

### ISSUE THREAD

**Issue by Sakurann** [oauth-wg/oauth-sd-jwt-vc#258]
Would also be good to add rationale why chosen certain claims are not selectively disclosable.

**Comment by Sakurann** [comment]
also what about `extends` and `extends#integrity` claims?

**Comment by danielfett** [comment]
Right, `vct#integrity` should be added/mentioned here: https://drafts.oauth.net/oauth-sd-jwt-vc/draft-ietf-oauth-sd-jwt-vc.html#section-3.2.2.2

`extends` and `extends#integrity` are not in the credential, so are not affected.

**Comment by bc-pi** [comment]
this text is/was about application profiles (like SD-JWT VC) saying what claims the issuer can make selectively disclosable and maybe has text that could serve as inspiration https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-12.html#name-selectively-disclosable-val

**Comment by bc-pi** [comment]
I think I maybe meant for that https://github.com/oauth-wg/oauth-sd-jwt-vc/issues/258#issuecomment-2400607967 to go on #257 but maybe works here too
